cmd/uhc/cluster/cmd.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright (c) 2019 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/openshift-online/uhc-cli/cmd/uhc/cluster/describe"
+	"github.com/openshift-online/uhc-cli/cmd/uhc/cluster/list"
+	"github.com/openshift-online/uhc-cli/cmd/uhc/cluster/status"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	debug   bool
+	cluster string
+}
+var Cmd = &cobra.Command{
+	Use:   "cluster COMMAND",
+	Short: "Get information about or login to a cluster",
+	Long:  "Get status, information, logs or a temporary login token for a cluster",
+	Run:   run,
+}
+
+func init() {
+	flags := Cmd.Flags()
+	flags.BoolVar(
+		&args.debug,
+		"debug",
+		false,
+		"Enable debug mode.",
+	)
+	Cmd.AddCommand(list.Cmd)
+	Cmd.AddCommand(status.Cmd)
+	Cmd.AddCommand(describe.Cmd)
+}
+
+func run(cmd *cobra.Command, argv []string) {
+	//check there is at least one argument
```

Comment:
I know it isn't important, but I would like to have an uniform style for comments in this project. I like them to start with an space after the comment mark, then a sentence starting with upper case, and ending with puntuation, usually a colon or a dot. For example:

```
// Check there is at least one argument:
```

Would you be so kind to do that?

Reply:
:+1: